metabase/metabase
# Remove unsafe-eval from script-src

Reverts metabase/metabase#33164
Related to https://github.com/metabase/metabase-private/issues/73

Un-reverts the removal of `unsafe-eval` in the `script-src` CSP header. Previously there were 2 issues:
- `ttag` still used `new Function` with plurals sometimes so I had to upgrade the library further
- trend lines were broken

How to verify:
- Open a line chart and add a trend line in the viz settings - it should work
- Go to a collection with line charts - the page should be displayed correctly

<img width="1376" alt="Screenshot 2023-08-15 at 12 00 15" src="https://github.com/metabase/metabase/assets/8542534/cfe56dd7-8823-46e9-976e-6ef1a83ab132">
